docs/software/gaming.md

```diff
@@ -2,16 +2,337 @@
 
-- [OpenTyrian - Gaming](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5#p45)
-- [Cuberite - Fast Minecraft server with web interface](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=60#p2068)
-- [MineOS - Multiple Minecraft servers with web interface](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=60#p2069)
-- [Nukkit - Server for Minecraft Pocket Edition](https://dietpi.com/phpbb/viewtopic.php?p=10675#p10675)
-- [Amiberry - Amiga emulation system, further developed optimised builds of uae4arm-rpi](https://dietpi.com/phpbb/viewtopic.php?p=64#p64)
-- [DXX-Rebirth - Descent 1 & 2 OpenGL port](https://dietpi.com/phpbb/viewtopic.php?p=2963#p2963)
-- [Steam - Steam client](https://dietpi.com/phpbb/viewtopic.php?p=8016#p8016)
+    ```
+    dietpi-software
+    ```
+
+    Choose **Software Optimised** and select one or more items. Finally click on `Install`. DietPi will do all the necessary steps to install and start these software items.
+
+    ![DietPi software](../assets/images/dietpi-software.jpg)
+
+    To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools) section.
+
+[Return to the **Optimised Software list**](../../dietpi_optimised_software)
+
+## OpenTyrian - Gaming
+
+Play a classic addictive game from the console or desktop.
+
+![DietPi gaming software OpenTyrian](../assets/images/dietpi-software-games-opentyrian.jpg){: style="width:500px"}
+
+=== "Start the game"
+
+    If you did not select OpenTyrian to start from boot:
+
+    - Console = OpenTyrian can also be run by typing `opentyrian`.
+    - Desktop = Use the OpenTyrian icons available in the start menu and desktop.
+
+    You can modify the AutoStart options in `dietpi-autoStart`.
+
+=== "Personal Note"
+
+    Tyrian (OpenTyrian) if not the best game in the world, is the best top down shooter/scroller game ever created.  
+    OpenTyrian is best experienced by using a mouse and the Enter key to change rear fire mode.  
+    Its old, retro, and a classic etc, but I doubt you will to find a recent and similar game that is anywhere close to OpenTyrian addictiveness.
+
+## Cuberite - Fast Minecraft server with web interface
+
+Cuberite allows you to create a single, blazingly fast Minecraft server, which utilizes the performance benefits of C++ (instead of Java). Also comes with a handy web interface.
+
+![DietPi gaming software Cuberite](../assets/images/dietpi-software-games-cuberite.png){: style="width:500px"}
+
+=== "Access the web interface"
+
+    - URL= `http://<your.IP>:1339` (replace with your DietPi's local IP)
+    - Username = `admin`
+    - Password = your global password
+
+=== "Tweaking"
+
+    Tweak the server settings by modifying the following files: (yes we recognised the typo which will be fixed with DietPi v6.33 ;))
+
+    - general server settings
+
+        ```bash
+        /mnt/dietpi_userdata/cubrite/settings.ini
+        ```
```

Comment:
This is no bash code but only a path, so let's not define bash language here?
Generally, I think everywhere where we say "bash", we mean "sh", since those cases do not use any bash-only syntax but can be used in any bourne-shell compatible shell like dash or csh or zsh etc 🙂. But it really doesn't matter as for such short commands there is anyway no syntax highlight 😄.

Reply:
Ok.